rotors_gazebo_plugins/src/gazebo_motor_model.cpp

```diff
@@ -98,13 +104,71 @@ void GazeboMotorModel::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf) {
     gzerr << "[gazebo_motor_model] Please specify a turning direction ('cw' or "
              "'ccw').\n";
 
+  if (_sdf->HasElement("motorType")) {
+    std::string motor_type = _sdf->GetElement("motorType")->Get<std::string>();
+    if (motor_type == "velocity")
+      motor_type_ = motor_type::VELOCITY;
+    else if (motor_type == "position")
+      motor_type_ = motor_type::POSITION;
+    else if (motor_type == "force") {
+      motor_type_ = motor_type::FORCE;
+    } else
+      gzerr << "[gazebo_motor_model] Please only use 'velocity', 'position' or "
+               "'force' as motorType.\n";
+  } else {
+    gzwarn << "[gazebo_motor_model] motorType not specified, using velocity.\n";
+    motor_type_ = motor_type::VELOCITY;
+  }
+
+  // setup joint control pid to control joint
+  if (_sdf->HasElement("joint_control_pid")) {
+    sdf::ElementPtr pid = _sdf->GetElement("joint_control_pid");
+    double p = 0;
+    if (pid->HasElement("p"))
+      p = pid->Get<double>("p");
+    double i = 0;
+    if (pid->HasElement("i"))
+      i = pid->Get<double>("i");
+    double d = 0;
+    if (pid->HasElement("d"))
+      d = pid->Get<double>("d");
+    double iMax = 0;
+    if (pid->HasElement("iMax"))
+      iMax = pid->Get<double>("iMax");
+    double iMin = 0;
+    if (pid->HasElement("iMin"))
+      iMin = pid->Get<double>("iMin");
+    double cmdMax = 0;
+    if (pid->HasElement("cmdMax"))
+      cmdMax = pid->Get<double>("cmdMax");
+    double cmdMin = 0;
+    if (pid->HasElement("cmdMin"))
+      cmdMin = pid->Get<double>("cmdMin");
+    pids_.Init(p, i, d, iMax, iMin, cmdMax, cmdMin);
+  } else {
+    pids_.Init(0, 0, 0, 0, 0, 0, 0);
```

Comment:
Maybe add an error if this is the case, as if this happens, it will not work (maybe even a fatal break would make sense here, or wdyt?)